chronos/apps/http/httpserver.nim

```diff
@@ -555,14 +555,16 @@ proc consumeBody*(request: HttpRequestRef): Future[void] {.
   finally:
     await reader.closeWait()
 
-proc getAcceptInfo*(request: HttpRequestRef): Result[AcceptInfo, cstring] =
+proc getAcceptInfo*(request: HttpRequestRef): Result[AcceptInfo, cstring] {.
```

Comment:
in general, functions returning `Result` should not raise excpetions - in this particular case, you should trace the call chain down to where the exception is being raised and catch it, translating it to a string instead.

That said, probably you have out-of-date dependencies - this error is not happening in CI

Reply:
> That said, probably you have out-of-date dependencies - this error is not happening in CI
Hmm
I did the following steps:
- git clone 
- nimble install
- nimble setup
- nim c chronos/apps/http/httpserver.nim

Reply:
Nimble dependency resolution is broken - you can often get it to work by removing its cache (`rm -rf $HOME/.nimble`) cc @zah

Reply:
Oddly it helped. Ty!